### PR TITLE
LON-1744 - allows per-polygon overriding of opacity

### PR DESCRIPTION
**Pull request by RileyOSL**
Set fillOpacity in options and :boom:

### CODE REVIEW

File: src/directives/polygons.js [hunk 1]
```diff
@@ -76,7 +81,12 @@ angular.module('ngMaps')
 
               // create polygon options with set opacity
               var opts = $scope.options ? $scope.options(c, properties, i, map) : {};
-              opts.fillOpacity = $scope.opacity ? $scope.opacity/100 : 1;
+              // allows per-polygon override of fillOpacity set via options
+              if(!opts.fillOpacity) {
+                opts.fillOpacity = $scope.opacity ? $scope.opacity/100 : 1;
+              } else {
+                opts.fillOpacity = opts.fillOpacity/100;
```
Comment (danielanteloagra) [inline src/directives/polygons.js]: repeated code... should be refactored into a method perhaps

Comment (RileyOSL) [inline src/directives/polygons.js]: 'tis indeed. I was wondering how far we should go with refactoring this... but i guess if we're in we're all in!

Comment (danielanteloagra) [inline src/directives/polygons.js]: up to you! @webignition would usually point these out, so in his absence i thought i would too :)

Comment (RileyOSL) [inline src/directives/polygons.js]: He sure would!

Yeah, We should probably take ownership of this fully now. I'll pretty it up a bit.

**Comment by danielanteloagra** [comment]
Nice!

**Comment by RileyOSL** [comment]
Tidied up as per our jslinting standards so view https://github.com/Overseas-Student-Living/ng-maps/pull/4/files?w=1 to see changes ignoring whitespace.

**Pull request merged**